spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
 end
 
 require "webmock/rspec"
+require "climate_control"
```

Comment:
I could go either way but I think because climate control doesn't have any config, I wouldn't include it in the spec helper.

Not a big deal either way.